docs/integrations/spark.md

```diff
@@ -21,14 +21,10 @@ redirect_from:
 There are two ways to use lakeFS with Spark:
 
 * Using the [lakeFS Hadoop FileSystem](#use-the-lakefs-hadoop-filesystem): Highly scalable, data flows directly from client to storage.
-* Using the [S3 gateway](#use-the-s3-gateway): Scalable and simpler to set up.
+* Using the lakeFS [S3-compatible API](#use-the-s3-compatible-api): Scalable and simpler to set up.
```

Comment:
Ordering remains strange: I'd expect "simpler to set up" to be the first option.

Reply:
Agreed, it's a debate- not touching it right now.